crates/release-automation/README.md

```diff
@@ -52,15 +58,37 @@ Automated steps still require running the tool manually ;-).
    Run a terminal in the root of the repository.
    Make sure you're on the commit from which you want to release.
 
-0. _[A]_ Create the release branch and bump the versions. In detail:
+0. _[M]_ Open a temrinal and prepare the environment:
+
+    ```sh
+    export HOLOCHAIN_URL="git@github.com:holochain/holochain.git"
+    export HOLOCHAIN_NIXPKGS_URL="git@github.com:holochain/holochain-nixkpgs.git"
+    export HOLONIX_URL="git@github.com:holochain/holonix.git"
+
+    export HOLOCHAIN_REPO=$(mktemp -d)
+    export HOLOCHAIN_NIXPKGS_REPO=$(mktemp -d)
+    export HOLONIX_REPO=$(mktemp -d)
+    ```
+
+0. _[M]_ Prepare the holochain repository and enter it.
+
+    ```sh
+    git clone "${HOLOCHAIN_URL}" "${HOLOCHAIN_REPO}"
+    pushd "${HOLOCHAIN_NIXPKGS_REPO}"
```

Comment:
this must be `pushd "${HOLOCHAIN_REPO}"`

Reply:
thanks!